collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -172,15 +172,8 @@ public void setExternalDataManager(ExternalDataManager externalDataManager) {
 
     public String getVersionedAppName() {
         String versionName = "";
```

Comment:
You can replace the three starting lines of the method with `String versionName = BuildConfig.VERSION_NAME.replaceFirst("-", "\n");`